src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/HttpConfiguration.java

```diff
@@ -44,4 +51,30 @@ private CloseableHttpClient getHttpClient() {
             .setDefaultRequestConfig(config)
             .build();
     }
+
+    @Bean
+    @ConditionalOnProperty(prefix = "storage", name = "proxy_enabled", havingValue = "true")
```

Comment:
with proxy, dunno - false or true occasion for `match if missing`?

Reply:
I think we do not need proxy. default value is false in property. for now I am leaving it like this.